submissions/Nik3264/Frogger-Game/app.js

```diff
@@ -0,0 +1,186 @@
+
+const gameProperties = {
+  cellHeight: 83,
+  cellWidth: 100,
+  playerXStart: 200,
+  playerYStart: 383,
+  up: 0,
+  down: 383,
+  left: 0,
+  right: 400,
+  collisionX: 50,
+  collisionY: 40,
+  frontierForEnemiesLeft:-150,
+  frontierForEnemiesRight:500,
+};
+
+class Game {
+    constructor({
+      cellHeight,
+      cellWidth,
+      playerXStart,
+      playerYStart,
+      up,
+      down,
+      left,
+      right,
+      collisionX,
+      collisionY,
+      frontierForEnemiesLeft,
+      frontierForEnemiesRight
+    }) {
+      this._level = 1;
+      this.isStop = false;
+      this.isGameOver = false;
+      this.CELL_HEIGHT = cellHeight;
+      this.CELL_WIDTH = cellWidth;
+      this.PLAYER_X_START = playerXStart;
+      this.PLAYER_Y_START = playerYStart;
+      this.up = up;
+      this.down = down;
+      this.left = left;
+      this.right = right;
+      this.collisionX = collisionX;
+      this.collisionY = collisionY;
+      this.frontierForEnemiesLeft=frontierForEnemiesLeft;
+      this.frontierForEnemiesRight=frontierForEnemiesRight;
+    }
+  
+    setLevel() {
+      this._level = 1;
+    }
+    getLevel() {
+      return this._level;
+    }
+    incLevel() {
+      this._level++;
+    }
+    gameStart() {}
+  }
+  
+class Enemy {
+  constructor({ x, y, speed }) {
+    this.sprite = "images/enemy-bug.png";
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+  }
+
+  update(dt) {
+    this.x +=
+      dt * (this.speed + 0.4 * this.speed * game.getLevel()) * !game.isStop;
+    if (this.x > game.frontierForEnemiesRight) {
+      this.x = game.frontierForEnemiesLeft;
+    }
+    if (this.isCollision()) {
+      player.start();
+      game.setLevel();
+    }
+  }
+
+  isCollision() {
+    if (
+      this.x > player.x - game.collisionX &&
+      this.x < player.x + game.collisionX &&
+      this.y > player.y - game.collisionY &&
+      this.y < player.y + game.collisionY
+    ) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Redundant.
`if`'s condition resolves to either `true` or `false` and based on that `then` or `else` part of `if` is executed.
You may simply return the values testing condition here.

Reply:
it was so obvious, but I didn't notice it! Thank you